it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerISpec.scala

```diff
@@ -1333,10 +1355,29 @@ trait BusinessIdentificationControllerISpec extends BaseISpec with SessionDataMi
       status(result) shouldBe 303
       redirectLocation(result) shouldBe Some(routes.StartController.showCannotCreateAccount().url)
 
-      verifyAgentAssuranceAuditRequestSentWithClientIdentifier(Utr("4000000009"), false, "SA6012", agentAssurancePayeCheck)
+      verifyAgentAssuranceAuditRequestSentWithClientIdentifier(
+          Utr("4000000009"),
+          false,
+          "SA6012",
+          agentAssurancePayeCheck)
       metricShouldExistAndBeUpdated("Count-Subscription-InvasiveCheck-Failed")
     }
 
+    "successfully selecting ICannotProvideEitherOfTheseDetails" in {
```

Comment:
Worth moving this test into the `when` scope of the "redirect to /cannot-create account page" test at 1312 above?

Reply:
Its already in the `when` scope.